pytorch_lightning/loggers/wandb.py

```diff
@@ -490,6 +505,7 @@ def _scan_and_log_checkpoints(self, checkpoint_callback: "ReferenceType[ModelChe
                             "save_top_k",
                             "save_weights_only",
                             "_every_n_train_steps",
+                            "_every_n_val_epochs",
```

Comment:
where does this come from? can't find any such attribute in model checkpoint callback.

Reply:
My mistake!